os/kernel/Kconfig

```diff
@@ -153,6 +153,7 @@ endif # !SCHED_TICKLESS
 
 config SYSTEM_TIME64
 	bool "64-bit system clock"
+	depends on LIBC_LONG_LONG
```

Comment:
Let's use `select`. Because it does not have any other dependency so that `select` gives better usability.